cylc/rose/stem.py

```diff
@@ -17,24 +17,20 @@
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
-"""rose stem [path]
+"""rose stem [path] [options]
 
-Install a rose-stem suite using cylc install.
+Rose stem:
+- Allows users to specify source trees to install using --source.
+- Provides Cylc install with standard jinja2 variables
+  These variables allow suite designers to group tasks for users to
+  specify with --group/--task.
```

Comment:
I'm not sure this is clearer, would prefer a concise description of the command followed by an overview o `--option`s.

```suggestion
Install a rose-stem suite using cylc install.
```

Reply:
I'd suggest:

Install a rose stem workflow with a specified set of source tree(s).
This command acts as a wrapper to "cylc install" by defining a number of additional Jinja2 variables.

The path to the workflow source directory can be specified using PATH.
Otherwise it must be provided in a directory named "rose-stem" in the first source tree.

Reply:
I've gone with @dpmatthews suggestion - I think that it's worth emphasizing the non magic nature of rose-stem.